File: abc/110/b.js
function main(input) {
  const formatted_input = input.split('\n').map( x => x.split(' ').map( x => parseInt(x, 10) ) );


  const first = formatted_input[0];
  const N = first[0];
  const M =first[1]; 
  const X = first[2];
  const Y = first[3];

  const Xs = formatted_input[1];
  const Ys = formatted_input[2];

  var max_x = -101;
  for(var x of Xs){ max_x = Math.max(x, max_x);}
  max_x = Math.max(X, max_x);

  var min_y = 101;
  for(var y of Ys){ min_y = Math.min(y, min_y);}
  min_y = Math.min(Y, min_y);

  if(max_x < min_y){
    console.log('No War');
  }else{
    console.log('War');
  }

}

main(require('fs').readFileSync('/dev/stdin', 'utf8'));

File: abc/110/b.cpp
#include<bits/stdc++.h>

#define rep(i,n) for(int i = 0; i < (n); i++)
#define ll long long

using namespace std;

int main(){
  int n,m,x,y;
  cin>>n>>m>>x>>y;

  int maxx=x;
  rep(i,n){
    int t;cin>>t;
    maxx=max(maxx,t);
  }

  int miny=y;
  rep(i,m){
    int t;cin>>t;
    miny=min(miny, t);
  }

  if(maxx<miny){
    cout<<"No War";
  }else{
    cout<<"War";
  }
  
  return 0;
}
